server/serializers.py

```diff
@@ -68,7 +71,29 @@ def _camelize_value(v: Any) -> Any:
         return v
 
 
-JsonizeContext = namedtuple("JsonizeContext", ["user", "session", "locale_id"])
+JsonizeContext = namedtuple(
+    "JsonizeContext", ["user", "session", "locale_id", "localizers"]
+)
+JsonizeModuleContext = namedtuple(
+    "JsonizeModuleContext", ["user", "session", "locale_id", "module_id", "localizers"]
```

Comment:
Again, can we avoid `localizers`?

And if we can't, let's make them `application_localizer`, `module_localizer`. But I think we can.